docs/android/tips/lollipop_emulator.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Test on lollipop emulator"
+sidebar_label: "Lollipop emulator"
+---
+
+## Overview
+
+To test the app on an AVD (Android Virtual Device) running Lollipop (Android API 21), you need to update the outdated WebView to ensure compatibility. Without this update, the WebView will crash.
```

Comment:
You do not use the abbreviation after introducing it, so don't introduce it. I also think most people will recognize it better if you just call it an emulator, like the page title.

```suggestion
To test the app on an Android emulator running Lollipop (Android API 21), you need to update the outdated WebView to ensure compatibility. Without this update, the WebView will crash.
```